apps/wallet-mobile/src/features/Exchange/useCases/CreateExchangeOrderScreen/CreateExchangeOrderScreen.tsx

```diff
@@ -51,7 +52,7 @@ export const CreateExchangeOrderScreen = () => {
   const quantity = `${amount.value}` as `${number}`
   const denomination = amountTokenInfo.decimals ?? 0
   const orderAmount = +Quantities.denominated(quantity, denomination)
-  const returnUrl = `${SCHEME_URL}${RAMP_ON_OFF_PATH}`
+  const returnUrl = `${SCHEME_URL}${configYoroiExchangeOrderShowCreateResult.path}?provider=${providerSelected}`
```

Comment:
can you confirm this @stackchain ?

Reply:
you can import the link factory --> linksYoroiMaker (if I'm not mistaken), initialize with `https` and it will provide all params

Reply:
yeah i used it to build the web !